packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.scss

```diff
@@ -12,8 +12,7 @@ $tf-typeahead-input-width: 135px !default;
 
 	border: 0;
 	border-radius: 0;
-	border-bottom: $form-border-bottom;
-	box-shadow: none;
+	box-shadow: $form-shadow;
```

Comment:
once in our webapp, I get 
```
Undefined variable: "$form-shadow".
          in /home/me/workspace/data-streams/data-streams/webapp/node_modules/@talend/react-forms/lib/widgets/MultiSelectTagWidget/MultiSelectTagWidget.scss (line 44, column 14)
```

Reply:
the variable is declares in packages/theme/src/theme/_forms.scss